.travis.yml

```diff
@@ -104,35 +165,13 @@ script:
           cat deps.log;
           echo "-- end of deps build log -----------------------------------------------";
           false; }
-    # compile / install Julia
-    - make $BUILDOPTS NO_GIT=1 prefix=/tmp/julia install | moreutils/ts -s "%.s"
-    - make $BUILDOPTS NO_GIT=1 build-stats
-    - du -sk /tmp/julia/*
-    - if [ `uname` = "Darwin" ]; then
-        for name in suitesparseconfig spqr umfpack colamd cholmod amd suitesparse_wrapper; do
-            install -pm755 usr/lib/lib${name}*.dylib* /tmp/julia/lib/julia/;
-        done;
-      fi
-    - cd .. && mv julia julia2
-    # run tests
-    - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
-        /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
-    - /tmp/julia/bin/julia -e 'versioninfo()'
-    - pushd /tmp/julia/share/julia/test
-    - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online download pkg
-    - popd
-    # test that the embedding code works on our installation
-    - mkdir /tmp/embedding-test &&
-        make check -C /tmp/julia/share/doc/julia/examples/embedding \
-             JULIA="DYLD_FALLBACK_LIBRARY_PATH='$DYLD_FALLBACK_LIBRARY_PATH' /tmp/julia/bin/julia" \
-             BIN=/tmp/embedding-test \
-             "$(cd julia2 && make print-CC)"
-    # restore initial state and prepare for travis caching
-    - mv julia2 julia &&
-        rm -f julia/deps/scratch/libgit2-*/CMakeFiles/CMakeOutput.log
```

Comment:
these files keep changing and needlessly making the cache update if we don't clear them out